release/ray_release/scripts/ray_bisect.py

```diff
@@ -108,7 +122,11 @@ def _bisect(
 
 
 def _sanity_check(
-    test: Test, passing_revision: str, failing_revision: str, run_per_commit: int
+    test: Test,
+    passing_revision: str,
+    failing_revision: str,
+    run_per_commit: int,
+    is_full_test: bool = False,
```

Comment:
nit: Do we need to set default values all the way down?

Reply:
I want to keep things explicit when I can, to make things less prompt for mistakes :D